### ISSUE TITLE
replace service switcher with a normal select (a11y)

### ISSUE THREAD

**Issue by phavekes** [SURFnet/sp-dashboard#1226]
[This issue is imported from pivotal - Originaly created at Jun 9, 2021 by Koen Cornelis](https://www.pivotaltracker.com/story/show/178469796)

	The service switcher utilises some kind of weird js-plugin to render a select.  The accessibility of the rendered html is not ok, and now necessitates a quick fix to remove the most dire problem.  This should be a simple select without a js-plugin.

Automatic testing identified 4 errors with it:
- inappropriate aria-role (see fix in a11y.ts)
- no name on the value of the dropdown (see fix in a11y.ts)
- it has no submit button(see fix in a11y.ts)
- the select has no label (see fix in a11y.ts)
- the textbox used is not a native element but a span with role=textbox.  This means we had to disable the prefers-native-element rule in the plugins/index.js file.

This is without actual testing in a screenreader or any other kind of testing.  Odds are there\'s more wrong with it.  More importantly it violates the first rule of aria: don\'t use aria if there\'s a native element which has the semantics for free.
